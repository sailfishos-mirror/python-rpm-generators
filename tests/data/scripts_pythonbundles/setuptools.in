packaging==16.8
pyparsing==2.2.1
ordered-set==3.1.1
tomli==1.2.3;python_version<"3.11"
