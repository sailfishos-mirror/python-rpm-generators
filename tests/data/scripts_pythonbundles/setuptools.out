Provides: bundled(python3dist(ordered-set)) = 3.1.1
Provides: bundled(python3dist(packaging)) = 16.8
Provides: bundled(python3dist(pyparsing)) = 2.2.1
